### PR TITLE
Sci mathematics/minisat: Unsilencing makefile + adding new version

### PR DESCRIPTION
**Pull request by g-braeunlich**

### CODE REVIEW

File: sci-mathematics/minisat/files/minisat-2.2.0_p20130925-nusmv.patch [hunk 1]
```diff
@@ -0,0 +1,874 @@
+diff -Naur minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543_ORIG/external/inttypes.h minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543/external/inttypes.h
+--- minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543_ORIG/external/inttypes.h	1970-01-01 01:00:00.000000000 +0100
++++ minisat-37dc6c67e2af26379d88ce349eb9c4c6160e8543/external/inttypes.h	2015-07-10 15:37:48.719358633 +0200
```
Comment (SoapGentoo) [inline sci-mathematics/minisat/files/minisat-2.2.0_p20130925-nusmv.patch]: @g-braeunlich could you try whether minisat still works without adding this `inttypes.h` file? GCC has native support for `inttypes.h`

Comment (g-braeunlich) [inline sci-mathematics/minisat/files/minisat-2.2.0_p20130925-nusmv.patch]: Yes it works without the inttypes.h header. The authors want to include this file for use with MSVC. It is not used / needed if we use gcc.

Comment (SoapGentoo) [inline sci-mathematics/minisat/files/minisat-2.2.0_p20130925-nusmv.patch]: @g-braeunlich understandable, but I don't like installing standard headers, lest they interfere with other headers. And even Gentoo Prefix on Windows/Interix won't use MSVC, so we can safely nix it.

File: sci-mathematics/minisat/minisat-2.2.0-r4.ebuild [hunk 2]
```diff
@@ -41,8 +42,8 @@ src_configure() {
 
 src_compile() {
 	export MROOT="${S}"
-	emake -C ${mydir} ${myconf}
-	LIB="${PN}" emake -C ${mydir} lib${myconf}
+	emake -C "${mydir}" "$myconf"
+	LIB="${PN}" emake -C ${mydir} lib"$myconf"
```
Comment (SoapGentoo) [inline sci-mathematics/minisat/minisat-2.2.0-r4.ebuild]: all the variables contain no whitespaces, any reason to change these?

**Pull request closed without merging**